project/src/com/google/daggerquery/server/Server.java

```diff
@@ -0,0 +1,73 @@
+package com.google.daggerquery.server;
+
+import com.google.common.collect.ImmutableList;
+import com.google.daggerquery.executor.QueryExecutor;
+import io.undertow.server.HttpServerExchange;
+import io.undertow.server.RoutingHandler;
+import io.undertow.Undertow;
+import io.undertow.util.HttpString;
+import io.undertow.util.StatusCodes;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.Deque;
+import java.util.stream.Collectors;
+
+public class Server {
+
+  private static final int port = 4921;
```

Comment:
As these are immutable constants I'd use UPPER_SNAKE_CASE. http://go/java-style#s5.2.4-constant-names

Reply:
Thank! I completely forgot to rename them.
Done ✅